cmd/dlt/cluster/cmd.go

```diff
@@ -85,6 +85,11 @@ func run(cmd *cobra.Command, _ []string) {
 
 	cluster := r.FetchCluster()
 
+	if args.bestEffort {
+		r.Reporter.Warnf("Deleting a cluster with 'best effort' means that certain resources may be left behind " +
```

Comment:
Maybe a prompt might be more effective :thinking:

Reply:
+1

Reply:
Makes sense, since we already have a prompt `delete cluster <cluster_key>?` I swapped the order, so the warning is displayed before the confirmation, WDYT?